oe_theme.theme

```diff
@@ -795,7 +796,23 @@ function oe_theme_preprocess_pattern_footer(array &$variables): void {
  */
 function _oe_theme_preprocess_link_types(array &$links): void {
   foreach ($links as &$link) {
-    $link['type'] = $link['link_type'];
-    unset($link['link_type']);
+    if (isset($link['link_type'])) {
+      $link['type'] = $link['link_type'];
+      unset($link['link_type']);
+    }
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_corporate_blocks_footer(array &$variables): void {
+  $variables['content'] = [
+    '#type' => 'pattern',
```

Comment:
We don't need this preprocess anymore after decision taken on #185, i.e. the footer is not an UI pattern anymore.

Reply:
Done